src/styles/components/cards.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  background-color: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__name {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    display: inline-block;
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+    width: 166px;
+
+    &-count {
+      text-align: right;
+      font-size: 10px;
+      font-style: normal;
+      color: #060b35;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    &-text {
+      color: #616070;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    background: #00acdc;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    cursor: pointer;
+  }
+
+  &__button:hover {
+    border-radius: 5px;
+    border-color: #00acdc;
+    background: #fff;
+    color: #00acdc;
+  }
```

Comment:
Let's do your transitions smooth. Also, border-radius is the same, so remove it from hover
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/e6e5c0b5-302e-40a3-8b00-0f9dd0016323)

Reply:
Comment about removing border-radius is not fixed